src/components/ServiceContainers/ServiceContainerLoading.tsx

```diff
@@ -68,7 +68,7 @@ export const ServiceContainerLoading = ({
   // (the component was rendered at some point before)
   //  - for example: when page index is changed from page 1 to page 2
   //  - this will make UI more smooth and it prevents flickering user experience
-  if (loading && data) return <RefreshStateCard>{children}</RefreshStateCard>;
+  if (loading && data) return <RefreshStateCard isInline={variant !== 'block'}>{children}</RefreshStateCard>;
```

Comment:
Put some comments here to describe the reason for this bug fix

Reply:
When `inline` variant of `ServiceContainerLoading` is used and data of the service container are reloaded, `block` variant of the loading bar was displayed. Instead, a small inline spinner needs to be shown.